deploy/app.yaml

```diff
@@ -7,7 +7,7 @@ instance_class: F1
 default_expiration: "30s"
 
 # (2) To get more memory and concurrent requests, a separate gunicorn is run.
-entrypoint: gunicorn -b :$PORT -w 1 --threads 2 --disable-redirect-access-to-syslog main:app
+entrypoint: gunicorn -b :$PORT -w 1 --threads 2 --disable-redirect-access-to-syslog --timeout 60 main:app
```

Comment:
Can you explain the reason for this timeout? Doesn't this now lead to tasks having a timeout of 60 seconds instead of 10 minutes?

Reply:
Oh sure. My laptop is sometimes a little slow. I got an annoying timeout. This helps, and is more equal to GAE (default timeout is 30 sec).

Reply:
Hmm I'm not sure if this influences the time behavior for tasks. We have to find out.

Reply:
In https://github.com/viur-framework/viur-app_server/pull/6 I added the timeout argument to this gunicorn launcher. Then it will use the default value of 60 seconds of the CLI (https://github.com/viur-framework/viur-app_server/blob/3d382cc00cc174586aa532164a0f0583fc5d4d9d/src/app_server/__init__.py#L325) anyway. And if you want you can increase this by providing a higher value (`viur run develop --timeout 360`)
Maybe this solves your issues. But, … @ArneGudermann has to release this change :D